SWLOR.Game.Server/Feature/ShipDefinition/NPCShipDefinition.cs

```diff
@@ -21,365 +21,629 @@ public class NPCShipDefinition : IShipListDefinition
 
         private void Tier1()
         {
-            _builder.Create("NPC_Sybil")
-                .ItemResref("npc_sybil")
-                .Name("NPC - Sybil")
-                .Appearance(AppearanceType.SWLORShipNeutralDropship)
```

Comment:
The appearance on all of these ships is missing. Can you double check on this? Are they actually showing the right models?

Reply:
The right models display, the appearance in the code seems to be unnecessary for npc's, it's just pc's that need it.